renderdoc/replay/replay_driver.h

```diff
@@ -37,6 +37,26 @@ struct FetchFrameRecord
   rdctype::array<FetchDrawcall> drawcallList;
 };
 
+struct GetTextureDataParams
```

Comment:
Very nice! More of the API needs this treatment honestly, but it's not a pressing concern and it can probably wait until the C# code is gone - then we have more flexibility by not being limited to what we can easily P/Invoke to/from the UI.